src/gui/transferlistfilterswidget.h

```diff
@@ -138,7 +133,7 @@ private slots:
 class CategoryFilterWidget;
 class TagFilterWidget;
 
-class TransferListFiltersWidget: public QFrame
+class TransferListFiltersWidget : public QFrame
```

Comment:
Here plural is correct since this widget is container for several filter widgets.